src/components/Settings/ShareOnSocialMedia.js

```diff
@@ -1,163 +1,88 @@
-import { TabHeading } from './SettingStyles';
+import _IconButton from '@material-ui/core/IconButton';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import {
+  FacebookShareButton as _FacebookShareButton,
+  LinkedinShareButton as _LinkedinShareButton,
+  TwitterShareButton as _TwitterShareButton,
+  FacebookIcon,
+  TwitterIcon,
+  LinkedinIcon,
+} from 'react-share';
+import styled, { css } from 'styled-components';
 
-const styles = {
-  buttonStyle: {
-    marginTop: '10px',
-    marginBottom: '0px',
-    fontSize: '15px',
-    fontWeight: 'bold',
-  },
-  raisedButtonStyle: {
-    margin: 20,
-    width: '240px',
-  },
-};
+const commonIconStyle = css`
+  margin-top: 1rem;
+  display: flex;
+  align-items: flex-start;
+  width: 100%;
+`;
 
-const classes = () => ({
-  twitter: {
-    backgroundColor: '#00aced',
-    '&:hover': {
-      backgroundColor: '#009CD5',
-    },
-  },
-  linkedin: {
-    backgroundColor: '#0077B5',
-    '&:hover': {
-      backgroundColor: '#006AA2',
-    },
-    facebook: {
-      backgroundColor: '#3B5998',
-      '&:hover': {
-        backgroundColor: '#354F88',
-      },
-    },
-  },
-});
+const ShareIconContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 0.7rem 2.5rem;
+`;
 
-const initFacebookAPI = () => {
-  window.fbAsyncInit = () => {
-    window.FB.init({
-      appId: '136831073699181',
-      cookie: true,
-      xfbml: true,
-      version: 'v2.11',
-    });
-    window.FB.AppEvents.logPageView();
-  };
+const TextContainer = styled.div`
+  height: 3rem;
+  line-height: 3rem;
+  verticalalign: center;
+  padding-left: 0.5rem;
+`;
 
-  ((d, s, id) => {
-    if (d.getElementsByTagName(s)[0]) {
-      let js,
-        fjs = d.getElementsByTagName(s)[0];
-      if (d.getElementById(id)) {
-        return;
-      }
-      js = d.createElement(s);
-      js.id = id;
-      js.src = 'https://connect.facebook.net/en_US/sdk.js';
-      fjs.parentNode.insertBefore(js, fjs);
-    }
-  })(document, 'script', 'facebook-jssdk');
-};
+const IconButton = styled(_IconButton)`
+  border-radius: 0.125rem;
+  padding: 0.5rem;
+  font-size: 1rem;
+`;
 
-const shareToFacebook = () => {
-  window.FB.ui(
-    {
-      method: 'share',
-      href: urls.CHAT_URL,
-      caption: 'SUSI by FOSSASIA',
-      hashtag: '#FOSSASIA',
-      quote: 'Lets chat with susi, the open source personal assistant',
-    },
-    response => {},
-  );
-};
+const FacebookShareButton = styled(_FacebookShareButton)`
+  ${commonIconStyle};
+`;
+
+const LinkedinShareButton = styled(_LinkedinShareButton)`
+  ${commonIconStyle};
+`;
 
-const ShareOnSocialMedia = props => {
-  initFacebookAPI();
-  const { raisedButtonStyle } = styles;
-  const { classes } = props;
+const TwitterShareButton = styled(_TwitterShareButton)`
+  ${commonIconStyle};
+`;
+
+const ShareOnSocialMedia = () => {
+  const shareUrl = 'https://susi.ai';
+  const title = 'Lets chat with susi, the open source personal assistant';
```

Comment:
Make it `Lets chat with SUSI.AI, the open source personal assistant`

Reply:
@akshatnitd done changes